pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -331,6 +331,11 @@ def on_response(self, response):
 
         For each message, schedule a callback with the executor.
         """
+        items = [
```

Comment:
Can you add a comment here that explains why we're doing this?

Reply:
done.